.azure-pipelines/hpu-tests.yml

```diff
@@ -37,8 +37,16 @@ jobs:
       displayName: 'Install dependencies'
 
     - bash: |
-         python ".azure-pipelines/run_hpu_tests.py"
-      displayName: 'HPU Tests in parallel'
+        python -m pytest -sv tests/accelerators/test_hpu.py --forked --junitxml=hpu1_test-results.xml
+      displayName: 'Single card HPU test'
+
+    - bash: |
+        python -m pytest -sv tests/accelerators/test_hpu.py --forked --hpus 8 --junitxml=hpu8_test-results.xml
```

Comment:
This would also a lot of tests twice which do not require the `hpus` param

Reply:
I'll parameterize it along with next change